### DISCUSSION TITLE
Any way to install without AWS/Google/Slack/et cetera?

### DISCUSSION THREAD

**Discussion by fooness** [outline/outline#1502] [General]
Outline looks very promising, but is there any way to install this on our servers **without** using AWS, Google, Slack, and whatnot … also, ideally, install everything **without** Docker.

Thank you in advance for clarifying and your advice.

**Comment by wlatic** [comment]
Check out:
https://github.com/outline/outline/issues/1491

I've installed this (and its working) on a VPS (google cloud server) running Centos and cyberpanel.net control panel.
